README.md

```diff
@@ -266,6 +267,18 @@ int main()
 }
 ```
 
+# Verifying the library
+
+## Static analysis using Clang
+
+The `clang_analyze.sh` Bash script in the root of the repository can be used to run the [Clang Static Analyzer](https://clang-analyzer.llvm.org) over the source code. It enables some extra checkers on top of the default ones. For the exact list of these checkers see the `enable_checkers` variable in the script. To run the static analysis, simply run
+
+```bash
+./clang_analyze.sh
+```
+
+from the repository root. After the analysis is done, follow the instructions printed to the terminal to view the resulting report.
```

Comment:
Looking at the script, running it from the repository root isn't strictly necessary right? Just as long as the path to the script is correct.

Reply:
True. I will rephrase it.